notebooks/09_Monitoring/09_01_Iris_Pipeline_Monitoring.ipynb

```diff
@@ -197,7 +197,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "iris_op = components.load_component_from_file('./src/09_01_Iris_Pipeline_Monitoring/component.yaml')"
+    "iris_op = components.load_component_from_file('./src/Iris_Pipeline_Monitoring/component.yaml')"
```

Comment:
Let's try to use lower_case for directory names

Reply:
Fixed.